buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy

```diff
@@ -29,10 +29,9 @@ class DownloadPage {
         new GuideGroup(title: "Install",
                 image: "${getImageAssetPreffix(url)}download.svg",
                 items: [
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installSdkman", title: 'Install with SDKman'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installHomebrew", title: 'Install with Homebrew'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installMacPorts", title: 'Install with MacPorts'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installWindows", title: 'Install through Binary on Windows'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install with SDKman'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install with Homebrew'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install through Chocolatey on Windows'),
                         new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#buildSource", title: 'Building from Source'),
```

Comment:
Current documentation doesn't contain anything related to build from source, so IMHO that should be fixed as well

Reply:
Is the "build from source" complete gone? It does not exist in `core` docs or in `starter`?

Reply:
@sdelamo didn't find at all

Reply:
I would say remove we need to remove the menu option or add the documentation back.

Reply:
Here is the [Build from source](https://docs.micronaut.io/1.3.7/guide/index.html#buildSource) documentation snippet in 1.3.7